custom/icds_reports/migrations/0036_allow_null_for_pnc_tables.py

```diff
@@ -3,6 +3,7 @@
 # flake8: noqa
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```

Comment:
Is this necessary to add for proper functionality or does it just make the modernize output a little noisier?

Reply:
It's unlikely that an autogenerated migration file would have any issues if this line were omitted.  The risk is that omitting that line lets someone write an import that works in py2 but fails in py3.  The risk seems higher in files written by hand.  In any case the modernize output is less noisy by fixing things like this.